stake_output.json

```diff
@@ -1 +1,12 @@
-{"testnet":{"deployInfo":{"codeId":870,"deployTimestamp":"Fri Mar 17 2023 14:05:12 GMT+0530 (India Standard Time)"},"instantiateInfo":{"contractAddress":"juno1q4lm5xhxqhx4d693qqwjqhyqlvzp9kvapmduuhkd87p92wt579nqedh43c","instantiateTimestamp":"Fri Mar 17 2023 14:05:13 GMT+0530 (India Standard Time)"}}}
\ No newline at end of file
+{
```

Comment:
Remove this, should only pick from the contracts artifacts, if not present then the UI should show that no contracts have been found to be deployed in artifacts and optionally show user the wasmkit compile and run script commans